backend/src/test/resources/testResultUpload/test-results-upload-valid-as-fhir.json

```diff
@@ -117,7 +117,7 @@
           "reference": "Patient/1234"
         },
         "effectiveDateTime": "2021-12-20T14:00:00Z",
-        "issued": "2023-05-24T19:33:06.472Z",
+        "issued": "2023-05-24T19:33:06Z",
```

Comment:
Is this just to have it in the same format as the other datetime fields that don't have millisecond precision?

Reply:
yes, definitely not necessary but seemed like it might be good to be more consistent and I don't _think_ millisecond precision is needed by our receivers anyway